packages/protocol/contracts/governance/Election.sol

```diff
@@ -845,4 +891,52 @@ contract Election is
     }
     return res;
   }
+
+  struct RevokeVotesInfo {
+    address[] groups;
+    uint256 remainingValue;
+  }
+
+  /**
+   * @notice Reduces the total amount of `account`'s voting gold by `value` by
+   *         iterating over all groups voted for by account.
+   * @param account Address to revoke votes from.
+   * @param value Amount of votes to revoke.
+   * @param lessers The groups receiving fewer votes than the i'th `group`, or 0 if
+   *                the i'th `group` has the fewest votes of any validator group.
+   * @param greaters The groups receivier more votes than the i'th `group`, or 0 if
+   *                the i'th `group` has the most votes of any validator group.
+   * @param indices The indices of the i'th group in the account's voting list.
+   */
+  function forceRevokeVotes(
+    address account,
+    uint256 value,
+    address[] calldata lessers,
+    address[] calldata greaters,
+    uint256[] calldata indices
+  ) external nonReentrant onlyRegisteredContract(LOCKED_GOLD_REGISTRY_ID) returns (uint256) {
+    require(value > 0 && value <= this.getTotalVotesByAccount(account));
```

Comment:
We don't usually use `this`

Reply:
Ok, I'll solve this by changing `getTotalVotesByAccount` to `public` instead of `external` which shouldn't make any noticeable difference I believe. 
The alternatives are to not call this fn (it isn't necessary anyway, you suggested it since it is more clear/efficient than just failing later), or to copy the fn's logic into this fn, which I don't like.
Let me know if `external` -> `public` has any issues, otherwise I'll do it that way.